src/Sulu/Bundle/ContactBundle/Entity/AccountRepository.php

```diff
@@ -188,7 +188,8 @@ public function findByIds($ids)
             ->addSelect('faxType')
             ->addSelect('notes')
             ->addSelect('medias')
-            ->where('account.id IN (:accountIds)');
+            ->where('account.id IN (:accountIds)')
+            ->orderBy('account.id', 'ASC');
```

Comment:
Interesting... Normally `DISTINCT` gives a big performance hit — `ORDER BY` not such — but if there's details here I'd love to see them. 

If order really doesn't matter then, yes, we can change the tests.